src/inverse/ws.rs

```diff
@@ -9,6 +9,9 @@ use std::time::Duration;
 use tungstenite::{connect, Message};
 use url::Url;
 
+const MAINNET: &str = "wss://stream.bybit.com/realtime";
+const TESTNET: &str = "wss://stream-testnet.bybit.com/realtime";
```

Comment:
Thankfully, there don't seem to be as many choices for inverse derivatives as there are for the spot websockets